Library/Homebrew/blacklist.rb

```diff
@@ -1,16 +1,5 @@
 def blacklisted? name
   case name.downcase
-  when 'screen', /^rubygems?$/ then <<-EOS.undent
```

Comment:
I definitely agree. Would `search_tap` in search.rb be the correct place for the new message?